indico/modules/rb_new/client/js/components/TimelineItem.jsx

```diff
@@ -69,27 +78,54 @@ export default class TimelineItem extends React.Component {
         return (segmentStartDuration - (startHour * 60)) / (step * 60);
     };
 
-    renderOccurrence = (occurrence, additionalClasses = '') => {
-        const {start_dt: startDt, end_dt: endDt, reservation, bookable} = occurrence;
+    renderOccurrence = (occurrence, additionalClasses = '', type = '') => {
+        let segmentStartDt, segmentEndDt, popupContent;
+        const {start_dt: startDt, end_dt: endDt, start_time: startTime, end_time: endTime,
```

Comment:
does eslint allow this? I'd rather go for something like this if it doesn't fit in a single line (hanging indent is not very common in JS):

```
const {
    ....
} = occurrence;
```